auth/router.js

```diff
@@ -0,0 +1,40 @@
+var express = require('express')
+var router = express.Router()
+var passport = require('passport')
+var signup = require('./signup')
+
+router.post('/signup', signup)
+
+router.post('/login', passport.authenticate('local'),
+  function(req, res) {
+    // TODO: check filed remeber and set Access-Control-Max-Age
+    res.status(200).json({
```

Comment:
1. I think that using `return` is a good practice. Consider http://stackoverflow.com/questions/16667783/express-route-return-necessary-or-not but primarily "Mistake #2" from https://www.toptal.com/nodejs/top-10-common-nodejs-developer-mistakes

2. the router here should only reference functions, defined in other files. Some of the routers will grow, and we want consistency throughout. So just like you used `signup`, there should be a `login` module, and every other endpoint should also reference imports.